bin/build_utils_gitlab

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash -e
+
+[[ "$TRACE" ]] && set -x
+
+CURRENT_DIR=$(cd $(dirname ${BASH_SOURCE[0]}); pwd)
+
+# source build utilities
+pushd ${CURRENT_DIR}
+  . ./build_utils
```

Comment:
Ditto: Why use the pushd instead of sourcing directly?

Reply:
this is useful in IDEs for being able to find dependencies in `./build_utils`. without this relative path most IDEs won't be able to resolve the deps